src/api.js

```diff
@@ -6,10 +6,18 @@ const myApi = axios.create({
     
 })
 
-export const getArticles = () => {
+export const getArticles = (topic_name) => {
     return myApi.get('/articles')
     .then((res) => {
-        return res.data.articles
+        console.log(res.data.articles, 'get res')
+        
+        if (!topic_name) {
```

Comment:
This is a cool solution, really like the check for undefined before trying to filter